CONTRIBUTING.md

```diff
@@ -5,3 +5,69 @@ We love improvements to our tools! EDGI has general [guidelines for contributing
 ## Submitting Web Monitoring Issues
 
 Issues that are project-wide, or relate heavily to the interaction between different components, should be added to our [Web Monitoring issue queue](https://github.com/edgi-govdata-archiving/web-monitoring/issues). Component-specific issues should be added to their respective repository.
+
+The following are recommended code styling and best practices for the web-monitoring-ui repository. We also have best practices for related to all the [web-monitoring] project (https://github.com/edgi-govdata-archiving/web-monitoring/blob/master/CONTRIBUTING.md) repo.
+
+## UI - Support
+### Browser Support:
+Last two of every major browser (Chrome, Safari, Firefox, Edge) except IE
+
+### ES6+ allowed everywhere
+ES6 and above is allowed. A feature is allowed if it has been formally published or is a Stage 4 proposal.  Otherwise, it is not allowed.
+
+## UI - Code Style / Best Practices
+
+### CSS Methodologies:
+Recommend BEM as much as possible (as opposed to (OOCSS, SMACSS, ITCSS)
+
+### Organization of React files:
+a) Constructor
+b) Lifecycle methods
+c) Public methods, usually passed as props to child components
+d) Render
+e) Additional render methods 
+f) Private 
+g) Utility functions outside of class definition these are useful only to the class so not worth pulling out into a file. They are not dependent on having a context. We neither use 'this' in the function, and calling 'this.utility' is wordy.
```

Comment:
Maybe a semicolon, em dash, or period between “definition” and “these?”

Reply:
Added.